src/Raven.Server/Web/System/StudioHandler.cs

```diff
@@ -152,9 +152,24 @@ public Task FavIcon()
         [RavenAction("/auth-error.html", "GET", AuthorizationStatus.UnauthenticatedClients)]
         public Task StudioAuthError()
         {
+            string reason;
+            string reasonAddon;
             var error = GetStringQueryString("err");
+
+            var stringBreak = error.IndexOf("<br>");
```

Comment:
can we find better solution for that? This is really fragile approach.

Reply:
do you mean use some pre-defined constant instead of ```"<br>"``` ?

Reply:
using `Environment.NewLine` instead